### DISCUSSION TITLE
Video screen magnified when watching on iOS

### DISCUSSION THREAD

**Discussion by dukyto** [ant-media/Ant-Media-Server#5163] [Q&A]
I insert iframe of antmedia server into my wordpress site, but when viewing on IOS device, it automatically zooms out the screen, how to fix it? hope everyone can help, thanks
![z4392169578428_438acb5e0071294453e2e5a3baae5d35](https://github.com/ant-media/Ant-Media-Server/assets/54923664/9141154a-b89e-4a68-b290-bc2be3c8dca0)

**Comment by mekya** [comment]
Hi @dukyto, 

I think it resolves if you add `playsinline` to your video tag as below. 

```
<video id="remoteVideo" autoplay controls playsinline ></video>
```

Let me know if it works. 

Regards
Oguz

**Reply by dukyto** [reply]
Okay thank you very much
Can you tell me how to not enlarge the screen when viewing on ios devices?

**Reply by mekya** [reply]
I think I miss the point. Could you explain further what you want to achieve?
